ui/webui/src/users/UserAccount.jsx

```diff
@@ -35,7 +84,14 @@ export const UserAccount = () => {
               title='Create user'
             />
             <PageSection>
-                Not implemented
+                <Form isHorizontal>
+                    <FormGroup label='Real name'>
+                        <TextInput id='gecos' onChange={onGecosChange} placeholder='John Doe' />
```

Comment:
You can just directly use setGecos state setter function instead of the wrapper that just calls this ;)